api/v1/db/oracledb/students-dao.js

```diff
@@ -19,14 +18,18 @@ const { getConnection } = appRoot.require('api/v1/db/oracledb/connection');
 const getResourceById = (id, sql, serializer, isSingleton, params) => new Promise(
   async (resolve, reject) => {
     const connection = await getConnection();
+    let term = params ? params.term : null;
     try {
-      let { rows } = await connection.execute(sql, [id]);
-      if (_.isEmpty(rows)) {
-        resolve(undefined);
-      } else if (isSingleton && rows.length > 1) {
+      if (term === 'current') {
```

Comment:
This is case sensitive. Is that your intent? It seems fine to me that would require "current" in lowercase.